example_project_app/plugins.py

```diff
@@ -275,3 +275,26 @@ def perform_project_modify(
                     action=action.lower(),
                 ),
             )
+
+    def validate_form_app_settings(
+        self, app_settings, setting_name, project=None, user=None
```

Comment:
There should not be a `setting_name` argument here. As specified before, this method should receive all of the app's app settings as one dict and validate one, several or all settings within that one method call. This allows us to e.g. validate multiple settings against each other, which is one of the use cases for this feature.